starboard/android/shared/install_target.gni

```diff
@@ -37,6 +37,10 @@ template("install_target") {
     deps = invoker.deps
     deps += [ ":$installable_target_name" ]
 
+    if (sb_is_evergreen_compatible) {
```

Comment:
This doesn't look quite right to me, because not all AOSP APKs actually use the loader app (e.g., the unit test APKs).

What problem is this solving?

Reply:
I think this fixed targets that we weren't supposed to be building on the loader platform. I've removed it from the OG PR and will remove it from here too. I think the duplicate symbol fix is the only actual code change we need.

Reply:
Removed